src/Nethermind/Nethermind.Trie/BatchedTrieVisitor.cs

```diff
@@ -293,10 +300,12 @@ private void BatchedThread()
 
                 cur.ResolveKey(_resolver, false);
 
-                SmallTrieVisitContext ctx = currentBatch[i].Item2;
+                SmallTrieVisitContext ctx = currentBatch[i].Item3;
```

Comment:
names these parameters in the ValueTuple and then use the name, so its not `.Item3`?

Reply:
Fixed.